library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/Mapping/Corporation.CorporationAbstract.orm.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping https://www.doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+  <mapped-superclass name="Ivoz\Provider\Domain\Model\Corporation\CorporationAbstract" table="corporation_abstract">
+    <indexes>
+      <index name="corporation_brandId" columns="brandId"/>
+    </indexes>
+    <unique-constraints>
+      <unique-constraint name="name_brandId" columns="name,brandId"/>
+    </unique-constraints>
+    <field name="name" type="string" column="name" length="255" nullable="false">
+      <options>
+        <option name="fixed"/>
+        <option name="default"/>
+      </options>
+    </field>
+    <field name="description" type="string" column="description" length="255" nullable="true">
+      <options>
+        <option name="fixed"/>
+        <option name="default">NULL</option>
```

Comment:
This seems to genearte 

```
  `description` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT 'NULL',
```
so it's not setting `NULL` but `'NULL'` as default value. 

Removing this line with default property will probably still add `DEFAULT NULL`  in the field delta.